views/partials/_navigation.njk

```diff
@@ -5,10 +5,10 @@ service navigation component. #}
 {%- set navLinks = [] %}
 {%- for item in navigation %}
   {%- set navLinks = (navLinks.push({
-    href: '/' + item.url + '/',
-    text: item.label,
-    current: true if permalink and (permalink == item.url) else false,
-    active: true if permalink and permalink.startsWith(item.url) else false
+    href: item.href,
+    text: item.text,
+    current: item.isCurrent(permalink),
+    active: item.isActive(permalink)
   }), navLinks) %}
 {%- endfor %}
```

Comment:
Would love for this to become something like a call to `navigation.topLevel(permalink)` that'd output data directly consumable by the Service Navigation rather than relying on the weird `set` inside the loop.

Reply:
Given we'll be keeping the accordion, we'll need that loop anyways so it's not really an issue that we need to call `item.isCurrent` or `item.isActive`.